.travis/build.sh

```diff
@@ -32,7 +32,8 @@ if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
   docker pull oryd/hydra:v1.0.0
   mvn test-compile spotbugs:check -e -V -B -f testsuite
   set +e
-  mvn -e -V -B clean install -f testsuite
+  mvn -e -V -B clean install -f testsuite -Pkafka-2_6_0
+  mvn -e -V -B test -f testsuite -Pkafka-2_5_0
   mvn -e -V -B test -f testsuite -Pkafka-2_4_1
   mvn -e -V -B test -f testsuite -Pkafka-2_4_0
```

Comment:
we removed these from the operator. should we still support them here?

Reply:
It does consume a bit more resources when running CI, but for the time being I see no reason to not keep supporting the older versions. People may also be using strimzi-kafka-oauth independently of strimzi-kafka-operator for their own cloud or stand-alone deployments. It's good to detect if something we do breaks compatibility with older versions so we can consider it and mention it in release notes once it happens.

Reply:
It makes sense, thanks.